src/backend/integrations/builtin/obs/effects/toggle-obs-source-filter.ts

```diff
@@ -76,13 +98,15 @@ export const ToggleSourceFilterEffectType: EffectType<EffectProperties> =
 
           $scope.searchText = "";
 
+          $scope.orphanedSources = [];
```

Comment:
I'd say, make this `missingSources` instead of `orphanedSources`, to clarify that sources aren't always orphaned but are missing for Firebot. (goes for all instances in this PR).

Reply:
changed